net_util/src/open_tap.rs

```diff
@@ -76,7 +76,14 @@ fn open_tap_rx_q_0(
     let tap = match if_name {
         Some(name) => Tap::open_named(name, num_rx_q, flags).map_err(Error::TapOpen)?,
         // Create a new Tap device in Linux, if none was specified.
-        None => Tap::new(num_rx_q).map_err(Error::TapOpen)?,
+        None => {
+            let tap = Tap::new(num_rx_q).map_err(Error::TapOpen)?;
+            log::info!(
+                "Created tap device: name={}, num_rx_q={num_rx_q}",
+                tap.if_name_as_str()
+            );
+            Tap::new(num_rx_q).map_err(Error::TapOpen)?
```

Comment:
Perhaps you meant to do the following?
```suggestion
            let tap = Tap::new(num_rx_q).map_err(Error::TapOpen)?;
            log::info!(
                "Created tap device: name={}, num_rx_q={num_rx_q}",
                tap.if_name_as_str()
            );
            tap
```

Reply:
yikes, a broken cherry-pick I assume. Sorry, thanks for catching this! It was very tough to do the cherry-picking, as there were so many conflicts

Reply:
@phip1611 does this mean I have to restart the long running tests?

Reply:
fixed